src/EditorFeatures/CSharpTest/ConvertTupleToStruct/ConvertTupleToStructTests.cs

```diff
@@ -96,7 +97,49 @@ public static implicit operator (int a, int b)(NewStruct value)
         return new NewStruct(value.a, value.b);
     }
 }";
-            await TestInRegularAndScriptAsync(text, expected, options: this.PreferImplicitTypeWithInfo(), testHost: host);
+            await TestAsync(text, expected, parseOptions: CSharp9, options: this.PreferImplicitTypeWithInfo(), testHost: host);
+        }
+
+        [Theory, CombinatorialData, Trait(Traits.Feature, Traits.Features.CodeActionsConvertTupleToStruct)]
+        public async Task ConvertSingleTupleTypeToRecord(TestHost host)
+        {
+            var text = @"
+class Test
+{
+    void Method()
+    {
+        var t1 = [||](a: 1, b: 2);
```

Comment:
Can you add a test with uppercase names? This request is related to https://github.com/dotnet/roslyn/issues/45451.

Reply:
In this case, all that is being added is the ability to generate a record struct. No change in logic is present to things like the Constructor generation.

Reply:
@CyrusNajmabadi The thing is, there should be a difference when the generating struct vs record struct.

The primary constructor parameter names should match the user preference for properties (usually pascal case), while for struct, it's usually camel case.

Reply:
To be more clear, when the names in the tuple are camel case, I personally don't care about the generated name casing. But when it's already Pascal Case, I think it should be preserved.

Reply:
This is not the purpose of this feature. It's too just get a simple conversion from a lightweight tuple to a corresponding record struct. I'm fine with follow-up work to expand on what this does. However, it will take more effort as it means fixing up call sites properly to reference the changed names. That's out of scope for this pr for me.

Reply:
> But when it's already Pascal Case, I think it should be preserved.

Sure. I'll make sure we do this.

Reply:
Ok done. :)